libraries/chain/steem_evaluator.cpp

```diff
@@ -2207,35 +2207,31 @@ namespace golos { namespace chain {
             });
         }
 
-        void decline_voting_rights_evaluator::do_apply(const decline_voting_rights_operation &o) {
-            database &_db = db();
-            FC_ASSERT(_db.has_hardfork(STEEMIT_HARDFORK_0_14__324));
-
-            const auto &account = _db.get_account(o.account);
-            const auto &request_idx = _db.get_index<decline_voting_rights_request_index>().indices().get<by_account>();
+        void decline_voting_rights_evaluator::do_apply(const decline_voting_rights_operation& o) {
+            const auto& account = _db.get_account(o.account);
+            const auto& request_idx = _db.get_index<decline_voting_rights_request_index>().indices().get<by_account>();
             auto itr = request_idx.find(account.id);
+            auto exist = itr != request_idx.end();
 
             if (o.decline) {
-                FC_ASSERT(itr ==
-                          request_idx.end(), "Cannot create new request because one already exists.");
-
-                _db.create<decline_voting_rights_request_object>([&](decline_voting_rights_request_object &req) {
+                if (exist) {
+                    GOLOS_THROW_OBJECT_ALREADY_EXIST("account", o.account);
```

Comment:
In this case we must say to client, that `decline_voting_rights` object already exists (not account).

Reply:
Forgot to change back after restoring tests. Fixed